platform/centec-arm64/linux-kernel-arm64.mk

```diff
@@ -0,0 +1,19 @@
+# linux kernel package for centec arm64
+
+export KVERSION= 4.9.62
+export LINUX_KERNEL_VERSION=4.9.62_4.9.62
```

Comment:
please use sonic kernel.

Reply:
4.9.62 is used by our released products.
we are working on migrating to 4.9.168 kernel.